WikiDBUpdaterHttp/WikiDBUpdaterHttp.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using Azure.Data.Tables;
+using WikiSentiment;
+
+
+namespace WikiDBUpdaterHttp
+{
+    /// <summary>
+    /// HTTP triggered function that updates DB with new wiki collections
+    /// </summary>
+    public class WikiDBUpdaterHttp
+    {
+        private ConfigurationWrapper config;
```

Comment:
Go read the Microsoft C# Coding Conventions. Your employer might have some specific company wide styleguide, but if not, you should have a good reason to not be be following [this](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions). (In general its full of details, but it specifically says private variables should be prefixed with underscore

```suggestion
        private ConfigurationWrapper _config;
```